pom.xml

```diff
@@ -86,13 +86,13 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-job</artifactId>
-            <version>2.9</version>
+            <version>2.11.1</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-cps</artifactId>
-            <version>2.27</version>
+            <version>2.33</version>
```

Comment:
Also tried `FlowNodeTest` with `workflow-cps` 2.32 (no `GraphListener`); `workflow-job` 2.9 (no `FlowExecutionListener`); and both together. Passed in all cases, albeit without optimization.